src/test/java/uk/gov/hmcts/reform/notificationservice/service/NotificationServiceTest.java

```diff
@@ -90,14 +90,21 @@ void should_mark_notification_as_failed_when_relevant_exception_from_client_is_c
     }
 
     @ParameterizedTest
-    @ValueSource(classes = {FeignException.NotFound.class, FeignException.InternalServerError.class})
+    @ValueSource(classes = {
+        FeignException.class,
+        FeignException.NotFound.class,
+        FeignException.InternalServerError.class
+    })
     void should_leave_notification_record_as_is_when_relevant_exception_from_client_is_caught(
         Class<FeignException> exceptionClass
     ) {
         // given
         var notification = getSampleNotification();
+        var exception = exceptionClass.getCanonicalName().equals(FeignException.class.getCanonicalName())
```

Comment:
Alternatively, we could just compare classes, i.e. `exceptionClass.equals(FeignException.class)`.

Reply:
derp, somehow i missed the fact ^